po/fr.po

```diff
@@ -1,177 +1,178 @@
 #
 msgid ""
 msgstr ""
 "Project-Id-Version: com.github.arshubham.cipher\n"
 "Report-Msgid-Bugs-To: \n"
 "POT-Creation-Date: 2019-01-31 09:23+0530\n"
 "PO-Revision-Date: 2019-01-30 23:41+0530\n"
-"Last-Translator: Automatically generated\n"
-"Language-Team: none\n"
+"Last-Translator: NathanBnm\n"
+"Language-Team: Français\n"
 "Language: fr\n"
 "MIME-Version: 1.0\n"
-"Content-Type: text/plain; charset=ASCII\n"
+"Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
 "Plural-Forms: nplurals=2; plural=(n > 1);\n"
 
 #: src/views/ROT13CipherView.vala:48 src/views/PolybiusSquareCipherView.vala:48
 #: src/views/CaesarCipherView.vala:53 src/views/Base64EncodingView.vala:48
 #: src/views/AtbashCipherView.vala:48 src/views/AsciiEncodingView.vala:48
 msgid "Encipher"
-msgstr ""
+msgstr "Chiffrer"
 
 #: src/views/ROT13CipherView.vala:49 src/views/PolybiusSquareCipherView.vala:49
 #: src/views/CaesarCipherView.vala:54 src/views/Base64EncodingView.vala:49
 #: src/views/AtbashCipherView.vala:49 src/views/AsciiEncodingView.vala:49
 msgid "Decipher"
-msgstr ""
+msgstr "Déchiffrer"
 
 #: src/views/ROT13CipherView.vala:51 src/views/PolybiusSquareCipherView.vala:51
 #: src/views/HashFunctionsView.vala:63 src/views/CaesarCipherView.vala:64
 #: src/views/Base64EncodingView.vala:51 src/views/AtbashCipherView.vala:51
 #: src/views/AsciiEncodingView.vala:51
 msgid "Plain Text"
-msgstr ""
+msgstr "Texte brut"
 
 #: src/views/ROT13CipherView.vala:54 src/views/PolybiusSquareCipherView.vala:54
 #: src/views/CaesarCipherView.vala:68 src/views/Base64EncodingView.vala:54
 #: src/views/AtbashCipherView.vala:54 src/views/AsciiEncodingView.vala:54
 msgid "Cipher Text"
-msgstr ""
+msgstr "Texte chiffré"
 
 #: src/views/HashFunctionsView.vala:47
 msgid "Generate Hash"
-msgstr ""
+msgstr "Générer un Hash"
 
 #: src/views/CipherView.vala:76
 #: data/com.github.arshubham.cipher.appdata.xml.in:55
 msgid "Caesar Shift Cipher"
-msgstr ""
+msgstr "Chiffre de César"
 
 #: src/views/CipherView.vala:77
 #: data/com.github.arshubham.cipher.appdata.xml.in:56
 msgid "Atbash Cipher"
-msgstr ""
+msgstr "Atbash"
 
 #: src/views/CipherView.vala:78
 #: data/com.github.arshubham.cipher.appdata.xml.in:60
 msgid "Polybius Square Cipher"
-msgstr ""
+msgstr "Carré de Polybe"
 
 #: src/views/CipherView.vala:79
 #: data/com.github.arshubham.cipher.appdata.xml.in:57
 msgid "ROT13 Cipher"
-msgstr ""
+msgstr "ROT13"
 
 #: src/views/CipherView.vala:80
 #: data/com.github.arshubham.cipher.appdata.xml.in:59
 msgid "Base64 Encoding"
-msgstr ""
+msgstr "Encodage Base64"
 
 #: src/views/CipherView.vala:81
 msgid "Hash Functions"
-msgstr ""
+msgstr "Fonctions de Hash"
 
 #: src/views/CipherView.vala:82
 #: data/com.github.arshubham.cipher.appdata.xml.in:58
 msgid "ASCII Encoding"
-msgstr ""
+msgstr "Encodage ASCII"
 
 #: src/views/CaesarCipherView.vala:61
 msgid "Number of letters to shift to the right: "
-msgstr ""
+msgstr "Nombre de lettres à décaler vers la droite : "
```

Comment:
Please remove the extra space.
```suggestion
msgstr "Nombre de lettres à décaler vers la droite: "
```

Reply:
In French the colon must be preceded by a space +1
For further information you can read this article. French punctuation has special rules : https://www.iwillteachyoualanguage.com/learn/french/french-tips/french-punctuation